Installer/asterics-are/package/linux/control

```diff
@@ -1,11 +1,756 @@
 Package: asterics-are
 Version: #{APPLICATION_VERSION_NUMBER}#
 Section: unknown
-Maintainer: Asterics Foundation
+Maintainer: AsTeRICS Foundation
+Upstream-Name: AsTeRICS
+Source: https://github.com/asterics/AsTeRICS
 Priority: optional
 Architecture: all
-Homepage: https://github.com/asterics/AsTeRICS
+Homepage: https://www.asterics.eu
 Provides: asterics-are
-Description: AsTeRICS Runtime Environment (ARE) - http://www.asterics.org
-Recommends: oracle-java8-installer
-Depends: libttspico-utils, libhidapi-libusb0, libudev1, xdg-utils
+Description: AsTeRICS Runtime Environment (ARE) - http://www.asterics.eu
+Recommends: oracle-java8-installer, libttspico-utils
+Depends: libhidapi-libusb0, libudev1, xdg-utils
+License: MIT or GPL-3 with Classpath exception
```

Comment:
do we need the whole license text here in the `control` file? wouldn't a link to the full text within the AsTeRICS Repository enough?

Reply:
I added it because during the installer build, there is a warning that the package did not have a license information. But it's still there although i added the license information. 

In this [commit](https://github.com/asterics/AsTeRICS/pull/330/commits/04ccaecf2f8b30faa4a040285178f719ad0b8978) I moved the inline license text to the copyright file and also the corresponding headers, but no success. I think I'll leave it like that now.